### ISSUE TITLE
TimeSerie bug - yyyy-mm not recognised as time

### ISSUE THREAD

**Issue by meyerch** [geosolutions-it/unesco-ihp#141]
Doing some tests wih TimeSerie, we came up with this error message:

![timeserie_timestamp_issue.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/59c90ce9c2355f0011ed2e90/125516c66e82c728ace21e0d46fc9d97269186bb86ec8057ad50f5873d472a63160db0386580a164fd52690e050a1de853524d41f5a8dd2fa3b26a26db5909e8876d5f77c63911adbd841dad1da862da11d1e9e1d30695aa2f2ff58a86453f1b071f8cf65b36a6904dca58e406.png)

Basically, it looks like the system does not recognize the time format, although it abides by the ISO 8601 rule (year / month --> yyyy-mm) 
We tried with other time details (year/month/day for instance in yyyy-mm-dd) and got the same error message. 

Please fix this issue, thanks.

**Comment by meyerch** [comment]
@afabiani  
I don't have the error message anymore but I still have an issue: 

The attribute table of the layer (http://ihp-wins-dev.geo-solutions.it/layers/geonode:TestMois) contains dates in the following format yyyy-mm . 

However, once uploaded on WINS DEV, those dates are transformed into yyyy-mm-ddThh:mm:ss as you can see in the screenshot below.

![timeserie_date.PNG](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/59c90ce9c2355f0011ed2e90/125516c66e82c728ace21e0d46fc9d97269186bb86ec8047a549f5d665043761260ea62864c7b96aea1b7a055b5a4afa17095a46f9af892ba5b638708b590defc3714d36976345f3e5db55ea43a82a994e82bcbd8f4ec7ee3a23e18b894736171755.PNG)

As a result, the time slider indicates "Wednesday, May 01, 1991" for instance instead of "May 1991". 

Please make sure that what is uploaded stays as is, i.e. the platform does not change the time format like this. Thank you
